### PR TITLE
Feature/giphy support ios binding

### PR DESCRIPTION
**Pull request by lhr000lhrmega**

### CODE REVIEW

File: bindings/Objective-C/MEGAChatSdk.mm [hunk 1]
```diff
@@ -855,6 +855,11 @@ - (void)attachNodesToChat:(uint64_t)chatId nodes:(NSArray *)nodesArray {
     self.megaChatApi->attachNodes(chatId, (nodeList != nil) ? [nodeList getCPtr] : NULL);
 }
 
+- (MEGAChatMessage *)sendGiphyToChat:(uint64_t)chatId srcMp4:(NSString *)srcMp4 srcWebp:(NSString *)srcWebp sizeMp4:(uint64_t)sizeMp4 sizeWebp:(uint64_t)sizeWebp  width:(int)width height:(int)height title:(NSString *)title {
+    MegaChatMessage *message = self.megaChatApi->sendGiphy(chatId, srcMp4 ? [srcMp4 UTF8String] : NULL, srcWebp ? [srcWebp UTF8String] : NULL, sizeMp4, sizeWebp, width, height, title ? [title UTF8String] : NULL);
```
Comment (jnavarrom) [inline bindings/Objective-C/MEGAChatSdk.mm:859]: Instead check if a string is null (using the operation ?) and pass UTF8String or NULL use directly NSString.UTF8String

```suggestion
    MegaChatMessage *message = self.megaChatApi->sendGiphy(chatId, srcMp4.UTF8String, srcWebp.UTF8String, sizeMp4, sizeWebp, width, height, title.UTF8String : NULL);
```

Comment (lhr000lhrmega) [reply bindings/Objective-C/MEGAChatSdk.mm:859]: Screen Shot 2020-10-21 at 8 10 21 PM
I followed the same coding style.
I am also ok with your suggested change.

Comment (jnavarrom) [reply bindings/Objective-C/MEGAChatSdk.mm:859]: Yes, this is what we did more than two years ago, but now we are using the other way. 

So, `string.UTF8String` is preferred over `string ? [string UTF8String] : NULL`

**Comment by sergiohs84** [comment]
Closing this PR. Already merged to the private repo

**Pull request closed without merging**